app/services/SSJ/invite_partner.rb

```diff
@@ -0,0 +1,19 @@
+class SSJ::InvitePartner < BaseService
+  def initialize(person_params, team, inviter)
+    @person_params = person_params
+    @team = team
+    @inviter = inviter
+  end
+
+  def run
+    person = Person.find_or_create_by!(email: @person_params[:email])
+    SSJ::TeamMember.create!(ssj_team: @team, person: person, role: SSJ::TeamMember::PARTNER, status: SSJ::TeamMember::INVITED)
+
+    unless user = User.find_by(person_id: person.id)
+      user = User.create!(email: person.email, person_id: person.id)
+    end
+    Users::GenerateToken.call(user)
+
+    PartnerMailer.invite(user, @inviter)
```

Comment:
Sorry, do you mind renaming this to `SSJMailer`.  I'm starting to think we should just have 1 mailer for all SSJ emails.  I should have done that with invite as well.